crates/ruma-api/src/error.rs

```diff
@@ -67,6 +67,17 @@ pub enum IntoHttpError {
     )]
     NeedsAuthentication,
 
+    /// Tried to create a request with an old enough version, for which no unstable endpoint
+    /// exists.
+    ///
+    /// This is also a fallback error for if the version is too new for this endpoint.
+    #[error("Tried to fall back to unstable path, but it did not exist for this endpoint.")]
```

Comment:
This should first mention something like "Endpoint not supported by server", then "and there is no unstable path to fall back to" or similar.